dockerfiles/cli/tests/cmd_backup_restore_tests.bats

```diff
@@ -0,0 +1,139 @@
+#!/usr/bin/env bats
+# Copyright (c) 2017 Codenvy, S.A.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#   Roman Iuvshyn
+
+source /dockerfiles/cli/tests/test_base.sh
+
+# Kill running che server instance if there is any to be able to run tests
+setup() {
+  kill_running_named_container che
+  remove_named_container che
+}
+
+teardown() {
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE stop --skip:nightly --skip:pull
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE destroy --quiet --skip:nightly --skip:pull
+}
+
+@test "test cli 'backup' command: backup fail if che is running" {
+  #GIVEN
+  if [ ! port_is_free 8080 ]; then
+    [ "$status" -eq 1 ]
+    [ "$output" = "Default port 8080 for che server is used. Cannot run this test on default che server port" ]
+  fi
+  tmp_path="${TESTRUN_DIR}"/cli_cmd_backup_fail_if_che_is_running
+  mkdir -p "${tmp_path}"
+  docker run --rm -v "${SCRIPTS_DIR}":/scripts/base -v /var/run/docker.sock:/var/run/docker.sock -v "${tmp_path}":/data $CLI_IMAGE start --skip:nightly --skip:pull
+  [[ "$(docker inspect --format='{{.State.Running}}' che)" == "true" ]]
+  ip_address=$(docker inspect -f {{.NetworkSettings.Networks.bridge.IPAddress}} che)
```

Comment:
we are using those 3 lines on many places. Maybe we could extract them and parametrize to make it reusable. WDYT?